webpack.config.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const webpack = require('webpack');
+const {CleanWebpackPlugin} = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  cache: true,
+  node: {
+    fs: 'empty', // Webex SDK `fs` dependency does not exist in browser.
+  },
+  devtool: 'inline-source-map',
+  resolve: {
+    extensions: ['.js', '.jsx'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+      {
+        test: /\.html$/,
+        use: [
+          {
+            loader: 'html-loader',
+          },
+        ],
+      },
+      {
+        test: /\.css$/,
+        use: ['to-string-loader', 'style-loader', 'css-loader'],
+      },
+      {
+        test: /\.scss$/,
+        use: ['to-string-loader', 'style-loader', 'css-loader', 'sass-loader'],
+      },
+      {
+        test: /\.(woff(2)?|ttf|eot|svg|png)(\?v=\d+\.\d+\.\d+)?$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              outputPath: 'assets/',
+            },
+          },
+        ],
+      },
+    ],
+  },
+  devServer: {
+    contentBase: path.resolve(__dirname, './demo'),
+    open: true,
+    overlay: true,
```

Comment:
Can you make this [show warnings](https://webpack.js.org/configuration/dev-server/#devserveroverlay) as well? Since we're going to use this while developing, we should make sure we also clear out warnings in our code